CDCgov/phdi
# Add design tweaks to eCR Viewer (Part 2)

# PULL REQUEST

## Summary
**Fixes & Tweaks:**
- eCR Summary - DOB
    - Fix how DOB is formatted in eCR summary
- Clinical Info
    - Fix the positioning of the Miscellaneous notes table
- No Data
    - Compile a list of example phrases that indicate that there is no data & filter sections out to Unavailable Data

## Related Issue
Fixes #1389 (Part 2)

## Screenshots
<img width="1469" alt="Screenshot 2024-04-17 at 12 37 06" src="https://github.com/CDCgov/phdi/assets/40042932/40650415-3287-4005-95ae-198322fe00cd">
<img width="1242" alt="Screenshot 2024-04-17 at 12 38 24" src="https://github.com/CDCgov/phdi/assets/40042932/03220f5e-1aa0-4433-a088-32c1b141e697">

## Checklist
- To verify the `No Data` text filtering:
    - Compare the version of the eCR Viewer for this [data](http://localhost:3000/view-data?id=1.2.840.114350.1.13.297.3.7.8.688883.567479) between this branch and the main branch.
    - Check that the following sections, as viewed in the main branch like below, are now present in the "Unavailable Data" section:
        - [ ] (Patient Info // Demographics) Race: Unknown
        - [ ] (Patient Info // Demographics) Ethnicity: Unknown
        - [ ] (Patient Info // Social History) Tobacco Use: Tobacco smoking consumption unknown
        - [ ] (Clinical Info // Clinical Notes) Miscellaneous Notes: Not on file documented in this encounter
        - [ ] (Clinical Info // Symptoms and Problems) Reason for Visit: Not on file

[//]: # (PR title: Remember to name your PR descriptively!)
